webapp/channels/src/components/integrations/installed_incoming_webhooks/installed_incoming_webhooks.tsx

```diff
@@ -14,7 +14,7 @@ import type {ActionResult} from 'mattermost-redux/types/actions';
 
 import BackstageList from 'components/backstage/components/backstage_list';
 import ExternalLink from 'components/external_link';
-import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+// import FormattedMarkdownMessage from 'components/formatted_markdown_message';
```

Comment:
don't comment out code, if it is not needed, delete it

Reply:
@Willyfrog I have deleted the comment and updated the code